### PR TITLE
Use system bridge

### PR DESCRIPTION
**Pull request by stucki**
This PR replaces #354 because it was incomplete.

### CODE REVIEW

**Comment by globin** [comment]
Could you check for virbr0, too, as a lot of us use libvirt to handle networking?

**Comment by stucki** [comment]
Sorry, don't use libvirt so I cannot test this. Anyway this should go into a new PR because it is not related to the LXC system bridge.

File: lib/vagrant-lxc/action/gc_private_network_bridges.rb [hunk 1]
```diff
@@ -35,8 +35,9 @@ def remove_bridges_that_are_not_in_use(env)
 
             if ! driver.bridge_is_in_use?(bridge)
               env[:ui].info I18n.t("vagrant_lxc.messages.remove_bridge", name: bridge)
-              # TODO: Output that bridge is being removed
-              driver.remove_bridge(bridge)
+              if bridge != "lxcbr0"
```
Comment (ccope) [inline lib/vagrant-lxc/action/gc_private_network_bridges.rb]: Please replace this with `if bridge not in ['lxcbr0', 'virbr0']`

**Comment by stucki** [comment]
Ok, done!

File: lib/vagrant-lxc/action/gc_private_network_bridges.rb [hunk 2]
```diff
@@ -35,8 +35,9 @@ def remove_bridges_that_are_not_in_use(env)
 
             if ! driver.bridge_is_in_use?(bridge)
               env[:ui].info I18n.t("vagrant_lxc.messages.remove_bridge", name: bridge)
-              # TODO: Output that bridge is being removed
-              driver.remove_bridge(bridge)
+              if bridge not in ['lxcbr0', 'virbr0']
```
Comment (fpletz) [inline lib/vagrant-lxc/action/gc_private_network_bridges.rb]: Sadly, Ruby is not Python.

This should work:
```ruby
unless ['lxcbr0', 'virbr0'].include? bridge
  driver.remove_bridge(bridge)
end
```

**Comment by stucki** [comment]
Oh my! Thanks for the note. Should have tested this :-)
Will come up with a fix in a second...

**Comment by stucki** [comment]
Works now. Thanks for your hint.

**Comment by fpletz** [comment]
Travis is green, code looks good, thanks! :+1:

This hack is okay for now, but at some point we should rethink how vagrant-lxc is doing networking. :)

**Comment by stucki** [comment]
Thank you!

**Comment by fgrehm** [comment]
> This hack is okay for now, but at some point we should rethink how vagrant-lxc is doing networking. :)

YES! :+1: :x: :100:

**Pull request merged**